app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
+
+  def new; end
+
+  def create; end
+
+  def destroy; end
```

Comment:
- [ ] Kindly remove these three controller actions. We only need `index` and `show` actions for the routes we defined.